static_src/test/unit/stores/service_plan_store.spec.js

```diff
@@ -137,6 +137,7 @@ describe('ServicePlanStore', function() {
 
     it('should set fetching to false, fetched to true', function() {
       ServicePlanStore.fetching = true;
+      ServicePlanStore.waitingOnRequests = false;
```

Comment:
It's so the service plan store can wait for all service plan requests to come through before setting fetching and fetched.